scripts/evrStatus

```diff
@@ -132,47 +129,23 @@ req_evrs_func() {
 	    fi
         done
     fi
-    if [[ "$hutch" == "TMO" ]]; then
-        req_evrs+=("IM3K4:PPM:CAM:EVR,")       # This PV needs to be added manually b/c the name does not have "$hutch" identifier in it
-
-    elif [[ "$hutch" == "XRT" ]]; then
-	req_evrs+=("HFX:DG2:BMMON:EVR,")       # This PV needs to be added manually b/c the name does not have "$hutch" identifier in it
-   
+    # These PVs need to be added manually b/c their names do not have a "$hutch" identifier in them
+    if [[ "$hutch" == "XRT" ]]; then
+	req_evrs+=("HFX:DG2:BMMON:EVR")
+	req_evrs+=("HFX:EVR:DG2:CVV:01")
+	req_evrs+=("HFX:EVR:MON:CVV:01")
     elif [[ "$hutch" == "FEE" ]]; then
-	req_evrs+=("IM1L1:PPM:CAM:EVR," 
-		   "IM1K1:PPM:CAM:EVR,"
-		  )       # This PV needs to be added manually b/c the name does not have "$hutch" identifier in it
-
-    elif [[ "$hutch" == "KFE" ]]; then
-	req_evrs+=("")       # This PV needs to be added manually b/c the name does not have "$hutch" identifier in it
+	req_evrs+=("IM1L1:PPM:CAM:EVR")
     fi
     export req_evrs
 }
 
-# Transforms array of requested PVs into an array of strings which is readable by the Extensible Display Manager
+# Displays requested and argument pvs
 to_display () {
-    local evr_num=${#req_evrs[@]} # counter for discerning which screen file to display
-
-    if [[ evr_num -lt 10 ]]; then
-        for ((i=0; i< evr_num; i++)); do  # assign EVR00=<HUTCH:EVR:PV0>, EVR01=<HUTCH:EVR:PV1>...
-            local macro_array+="EVR0$i=${req_evrs[$i]}"  # array containing macro assignments 
-        done
-
-    elif [[ evr_num -ge 10 ]]; then
-        for ((i=0; i< 10; i++)); do  # assign EVR00=<HUTCH:EVR:PV0>, EVR01=<HUTCH:EVR:PV1>...
-             local macro_array+="EVR0$i=${req_evrs[$i]}"  # array containing macro assignments 
-        done
-        for ((j=10; j < ${#req_evrs[@]}; j++ )); do
-            macro_array+="EVR$j=${req_evrs[$j]}"
-        done
-    fi
-    local macros_exported=${macro_array[*]} # macros are exported to edm as a string
-    export macros_exported evr_num
-#   echo $macros_exported
+    source /cds/group/pcds/pyps/conda/pcds_conda
     echo "Displaying EVR status screen for " "$hutch" "..."    
     printf "\n"           
-    
-    /reg/g/pcds/epics-dev/screens/edm/common/current/evrStatus/edm-evrs.cmd
+    pydm /reg/g/pcds/epics-dev/screens/pydm/scripts/evr.py -p "${req_evrs[@]}" "${pv_args[@]}" &
```

Comment:
I see the main change here is shelling out to a dev evr.py screen- is there any way this can be brought in for review too?

It's really confusing having these be separate entities and version controlled separately. It was strange in the previous version as well, to be fair- so it doesn't have to be your scope to correct this.

Is evr.py used outside the scope of this script?

Reply:
evr.py isn't used outside of this, I've integrated it